.gitignore

```diff
@@ -68,3 +68,7 @@ util/Win_Check_Output.txt
 secrets.tar
 id_rsa_*
 /.vs
+lib/lufa/Projects/LEDNotifier/CPUUsageApp/obj/Debug/CPUMonitor.csproj.CoreCompileInputs.cache
```

Comment:
It may be best to add these to `.git/info/exclude` instead, since this seems to be a local thing, only?

Reply:
sorry... I thought I was creating this pull request in my local fork.  Sorry about this. Please ignore.